app/controllers/admin/merchants_controller.rb

```diff
@@ -0,0 +1,38 @@
+class Admin::MerchantsController < ApplicationController
+  def index
+    @merchants = Merchant.top_5_merchants
```

Comment:
Nice use of the instance variables here and eliminating the use of calling the method on the view

Reply:
🙇🏻‍♂️